test/integration/targets/ansible-galaxy-collection-scm/tasks/empty_installed_collections.yml

```diff
@@ -1,7 +1,7 @@
 - name: delete installed collections
   file:
-    state: "{{ item }}"
-    path: "{{ galaxy_dir }}/ansible_collections"
+    state: absent
+    path: "{{ item }}"
   loop:
-    - absent
-    - directory
+    - "{{ install_path }}"
```

Comment:
I refactored the tests to use variables to make sure everything was getting cleaned up between tests since I added another collection directory (to compare the results installing from an artifact vs git repo).